tests/run_command_panel.py

```diff
@@ -92,7 +111,7 @@ def save(self, key, recents):
     def load(self, key):
         return {
             command: {"timestamp": datetime.datetime.now()}
-            for command, _, _, _, _, _, _ in list(commands)[0:3]
+            for command, _, _, _, _, _, _ in list(commands)[0:6]
```

Comment:
This line is very hard to read, I know you only upped a number, but I don't know why. It's picking the first six values out of a commands variable that I think was a dictionary  but is being cast to a list.

Reply:
and that contains 7 values to be unpack of which we only want the first one.

Reply:
feel free to ignore, but just reading this, it hard to understand in what way it is parsing the data structure.